charts/ChartData.ts

```diff
@@ -19,6 +19,7 @@ import { Color } from "./Color"
 import { ChartDimensionWithOwidVariable } from "./ChartDimensionWithOwidVariable"
 import { OwidSource } from "./owidData/OwidSource"
 import { entityName, entityId, entityCode } from "./owidData/OwidTable"
+import { xor } from "lodash"
```

Comment:
You'll have to import specifically in `Util.ts` and then import that here. That's also why bundlewatch complains.

Reply:
I've been meaning to ask why importing something from lodash in Utilt.ts, and exporting there, then imported from Util, enables tree shaking? Shouldn't tree shaking work the same in either case? Obviously I know there's some gotcha for why we have to do this, but I just forget.

Reply:
That's because `lodash` is not tree-shakeable, and we circumvent that by just importing its "submodules" like `lodash/xor` through `Util.ts`. That's the only way we can ensure we only import the methods we actually use.
I think the reason actual `lodash` is not tree-shakeable is that it doesn't export ES modules, and these are needed for proper tree shaking.

The actual solution would be to use `lodash-es` because that's tree-shakeable by default, but that then causes issues in conjunction with `jest` which is a whole other headache. This could get better with Jest 27 and Node 14, though, as both are supposed to have native ES module support then.